_clients/logstash/read-from-opensearch.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Read from OpenSearch
+parent: Logstash
+nav_order: 220
+---
+
+# Read from OpenSearch
+
+As we ship Logstash events to an OpenSearch cluster using the OpenSearch output plugin. We can also perform read operations on an OpenSearch cluster and load data into Logstash using the OpenSearch input plugin.
+
+The OpenSearch input plugin helps to read the search query results performed on an OpenSearch cluster and load them into Logstash. This is useful for replaying test logs, reindexing, etc. Users can schedule this ingestion to run periodically using 
+cron syntax(using `schedule` configuration setting) or by just running the query once to load data into Logstash.
+
+
+
+## OpenSearch input plugin
+
+To run the OpenSearch input plugin, add the configuration in your `pipeline.conf` file like the sample configuration shown below:
+
+```yml
+input {
+  opensearch {
+    hosts       => "https://hostname:port"
+    user        => "admin"
+    password    => "admin"
+    index       => "logstash-logs-%{+YYYY.MM.dd}"
+    query       => "{ "query": { "match_all": {}} }"
+  }
+}
+
+filter {
+}
+
+output {
+}
+```
+
+Using the above sample configuration, the `match_all` query filter is triggered and data is loaded once.
+
+schedule setting can be used to periodically schedule ingestion using cron syntax.
+
+Example: `schedule => "* * * * *"` Adding this to the above configuration loads the data every minute.
```

Comment:
I'm assuming this sentence means users can add a cron expression to the `input` section to ingest data? If so, we should explicitly specify the input section. Perhaps we can change this text to the following?

"To ingest data according to a schedule, use a cron expression that specifies the schedule you want. For example, to load in data every minute, add `schedule => "* * * * *"` to the input section of your `pipeline.conf` file.

Reply:
ack